torch/_inductor/decomposition.py

```diff
@@ -948,3 +949,17 @@ def max_pool2d_with_indices(
         padding,
     )
     return vals, indices
+
+
+# TODO: remove this when gcc 10 support is dropped in inductor
```

Comment:
Can you open an issue / link to an existing one for this? The TODO reads like there is a compiler bug in gcc 9/10, but the comment below reads like there is a bug in our codegen.